cli/src/command_error.rs

```diff
@@ -463,12 +467,14 @@ impl From<TempTextEditError> for CommandError {
     }
 }
 
+#[cfg(feature = "git")]
```

Comment:
nit: maybe we can extract these `impl`s to inner module, and add `#[cfg(feature = "git")]` there.